src/library/hiddentablemodel.cpp

```diff
@@ -1,34 +1,30 @@
 #include "library/hiddentablemodel.h"
 
+#include "library/dao/trackschema.h"
 #include "library/trackcollection.h"
 #include "library/trackcollectionmanager.h"
-#include "library/dao/trackschema.h"
-
 
 HiddenTableModel::HiddenTableModel(QObject* parent,
-                                   TrackCollectionManager* pTrackCollectionManager)
+        TrackCollectionManager* pTrackCollectionManager)
         : BaseSqlTableModel(parent, pTrackCollectionManager, "mixxx.db.model.missing") {
     setTableModel();
 }
 
 HiddenTableModel::~HiddenTableModel() {
 }
 
-void HiddenTableModel::setTableModel(int id) {
-    Q_UNUSED(id);
+void HiddenTableModel::setTableModel() {
     QSqlQuery query(m_database);
     const QString tableName("hidden_songs");
 
     QStringList columns;
     columns << "library." + LIBRARYTABLE_ID;
-    QString filter("mixxx_deleted=1");
+    // clang-format off
```

Comment:
Is this really necessary? How does it look without it?

Reply:
It breaks the line after every single "+", which makes it horribly unreadable

Reply:
Nevermind, found a way to work around it.